hoomd/state.py

```diff
@@ -162,10 +162,10 @@ class State:
     dihedrals, and impropers contain 2, 3, 4, and 4 particles per group
     respectively. Bonds specify the toplogy used when computing energies and
     forces in `md.bond`, angles define the same for `md.angle`, dihedrals for
-    `md.dihedral` and impropers for `md.improper`. These collectively implement
-    bonding potentials used in molecular dynamics force fields. Like bonds,
-    special pairs define connections between two particles, but special pairs
-    are intended to adjust the 1-4 pairwise interactions in some molecular
+    `md.dihedral` and impropers for `md.improper`. These collectively
+    implement bonding potentials used in molecular dynamics force fields. Like
+    bonds, special pairs define connections between two particles, but special
+    pairs are intended to adjust the 1-4 pairwise interactions in some molecular
```

Comment:
Why were these lines changed? If they were not too long before I don't know why they would be too long now.

Reply:
I had some issues getting `md.improper` to link and tried a few changes to these lines to fix it. Those changes turned out to not be needed, and I failed to undo the change to the wrapping.